style.css

```diff
@@ -169,3 +188,783 @@ ul {
   left: 0.013671875px;
   padding: 2px;
 }
+
+.works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 120px;
+  padding-bottom: 120px;
+  position: relative;
+  width: 375px;
+  height: 3455px;
+  top: 682px;
+  z-index: +5;
+}
```

Comment:
- [ ] [Required] The works section should be made to use the CSS declaration `display: grid;` instead of using flexbox. Let's try to fix this please to make sure we stick to the requirements and make our code look more professional.